src/createServer.js

```diff
@@ -1,11 +1,24 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+const { usersRouter } = require('./routes/users.route.js');
+const { expensesRouter } = require('./routes/expenses.route.js');
+const { resetUsers } = require('./services/users.services.js');
+const { resetExpenses } = require('./services/expenses.services.js');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  app.use(cors());
+
+  resetUsers();
+  resetExpenses();
+
+  app.use('/users', express.json(), usersRouter);
+  app.use('/expenses', express.json(), expensesRouter);
```

Comment:
```suggestion
  app.use(cors());
  app.use(express.json());

  resetUsers();
  resetExpenses();

  app.use('/users', usersRouter);
  app.use('/expenses', expensesRouter);
```
You can apply middleware to all routes at once